src/Futhark/CodeGen/Backends/PyOpenCL.hs

```diff
@@ -185,7 +185,7 @@ compileProg mode class_name prog = do
 -- We have many casts to 'long', because PyOpenCL may get confused at
 -- the 32-bit numbers that ImpCode uses for offsets and the like.
 asLong :: PyExp -> PyExp
-asLong x = Py.simpleCall "np.long" [x]
```

Comment:
This may be one case where we do need to use `int` instead, since this function is used for values that end up getting passed to PyOpenCL, which may not like Numpy types.  We'll see what the CI says.

Reply:
I've tried both and both seems to have passed the tests on my machine.